### ISSUE TITLE
Incorrect Damage Multiplier Calculation 

### ISSUE THREAD

**Issue by DoomLaser9** [cmss13-devs/cmss13#422]
## Description

The barrel charger produces inconsistent damage increases, and custom damage multipliers for map objects fail to apply.

## Difference between expected and actual behavior

The barrel charger should increase damage by 30% for non-shotguns. This works with the holdout pistol, but with the standard M41A, it only increases the damage to 56 from 44, when it should be 57.2.

It also only increases a standard 88 Mod 4's damage to 37.5 from 30, when it should be 39.

Also, M37-17's that have damage multipliers of 1.1 (e.g. certain `/obj/item/weapon/gun/shotgun/pump/cmb` instances in fiorina science annex map) don't actually show any damage increases at all.

## Steps to reproduce

Look at the in-game combat statistics of the different guns with and without barrel chargers.

An example of a special un-acidable M37-17 that's supposed to have a damage increase is located in the "electrician/engineering room" to the left of the marshal office on fiorina science annex, on a table.

## Issue bingo (not required to be all checked)

- [x] Issue could be reproduced at least once
- [x] Issue happened in a recent (less than 7 days ago) round
- [x] Couldn't find an existing issue about this (https://github.com/cmss13-devs/cmss13/issues)

**Comment by Diegoflores31** [comment]
for the case of mod 88 the issue is that mod 88 original ammo its suposed to make 25 damage but the Weapon itself multiplies the damage by 20% to a total of 30 ... now the thing is that the attachment multiplies the 25 of the bullet and then adds to the 20% of the gun making a total of 39 . this can be changed but it would need aproval because it will increase damage on this weapons by a lot and impact gameplay

**Comment by BKryth** [comment]
After checking the issue myself, there is actually no issue here, as the m41a mk2 itself has a 1.1 damage multiplier and the Barrel Charger adds a 0.3 damage multiplier, so this means that the gun does a damage multiplier with a total of 1.4. A standard m41a mk2 magazine uses 10x24 ammo which has 40 damage by default, which when put on a m41a mk2 without BC, does 44 damage due to the 1.1 damage multiplier

As for the 88mod, their bullet has 25 damage and the pistol itself has 1.2 damage multiplier, so with the added BC, it should have a 1.5  damage multiplier which would mean 37.5

Don't see any issue here, just misinterpretation on how damage is calculated.

tldr; no bug to fix unfortunate, no GBP to find here

**Comment by MidwayAtom** [comment]
As above, closing as no issue.
